common/reviews/api/rush-lib.api.md

```diff
@@ -1023,7 +1023,7 @@ export class ProjectChangeAnalyzer {
 export class RepoStateFile {
```

Comment:
Some thoughts about this problem:
- `commonRushConfigFolder` is still meaningful with subspaces, since it contains many config files that are unaffected by subspaces (**experiments.json**, **deploy.json**, etc).  We should not replace this API.
- Instead we could add a new API (alongside the old one) such as `getSubspaceConfigFolder(subspace)` for accessing subspace config. Where possible, the "subspace" parameter should NOT be optional, because optional parameters are very error prone (it's easy to forget to pass along a value)

- On the other hand, the `RushConfiguration.commonVersions` API does have a new meaning for subspaces.  We could handle it two ways:

   1. The RFC spec seems to say that the `"default"` subspace would have the file path **common/config/subpaces/default/common-versions.json**.  What about the old path **common/config/rush/common-versions.json**?  If subspaces are enabled, maybe we should forbid that file to avoid mistakes? Maybe the old `RushConfiguration.commonVersions` API should return an empty string or report an error?
   2. Another idea would be to make the `"default"` subspace special: Its file path is **common/config/rush/common-versions.json** and therefore APIs like `RushConfiguration.commonVersions` retain their old interpretation -- they simply refer to the default subspace.
   
I think maybe I favor option (ii).

Reply:
This was fixed in the new design